LoRaEngine/test/LoRaWanNetworkServer.Test/ConfigurationTest.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoRaWanNetworkServer.Test
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Text;
+    using LoRaWan.NetworkServer;
+    using Xunit;
+    using Xunit.Extensions;
+
+    public class ConfigurationTest
+    {
+        [Theory]
+        [MemberData(nameof(AllowedDevAddressesInput))]
+        public void Should_Setup_Allowed_Dev_Addresses_Correctly(string inputAllowedDevAddrValues, HashSet<string> expectedAllowedDevAddrValues)
+        {
+            Environment.SetEnvironmentVariable("AllowedDevAddresses", inputAllowedDevAddrValues);
```

Comment:
Cool, did not know about it. So I can keep overwriting the values to test multiple scenarios?

Reply:
Yes, the method/Class can provide different inputs. It's like different line of inlinedata, but here i can pass instantied objects as parameters, whereas inlinedata seems to only accept value types.

Reply:
Sorry, I did not formulated properly. I was talking about setting the environment variables.

Reply:
Yeah, Looks like it :)